app/controllers/application_controller.rb

```diff
@@ -62,4 +62,8 @@ def render_unauthorized
     headers['WWW-Authenticate'] = 'Token realm="Application"'
     render json: 'Not Authorized', status: 401
   end
+
+  rescue_from ActionController::ParameterMissing do
+    render json: { error: 'parameter missing' }, status: :bad_request
+  end
```

Comment:
https://github.com/department-of-veterans-affairs/va-api-common/blob/master/lib/va/api/common/exceptions/parameter_missing.rb